### DISCUSSION TITLE
make component when resource directory changed

### DISCUSSION THREAD

**Discussion by RicLzPt** [livewire/livewire#4022] [1. Help]
I changed my resource directory, moved its to app folder. But, when I run make component, livewire create again resource folder in root path instead in new directory defined.

**Comment by joshhanley** [comment]
@RicLzPt you will need to publish your Livewire config, and update `view_path` to point at your new resources directory location.

Hope this helps!

https://github.com/livewire/livewire/blob/1f5d3e3897b19f1bf94bf6e926b1fb70e41939a7/config/livewire.php#L20-L30
